client/app/l10n/en.json

```diff
@@ -52,7 +52,7 @@
   "Allow the whistleblower to add attachments to the submission": "Allow the whistleblower to add attachments to the submission",
   "Allow the whistleblower to send messages": "Allow the whistleblower to send messages",
   "Allow the whistleblower to write comments": "Allow the whistleblower to write comments",
-  "Allow to advance through submission steps only when current step is completed": "Allow to advance through submission steps only when current step is completed",
+  "Allow to advance through submission steps only when current step is completed": "Allow advancement of submission process only upon finishing current step",
```

Comment:
Removing completely the string as the feature has been never used so it could be removed as well.